synthetic_data/generators.py

```diff
@@ -166,7 +167,9 @@ def _generate_uncorrelated_column_data(self, num_samples):
                     )
                 )
             else:
-                if col_["order"] is not None:
+                if (not generator_name == "null_generator") and col_[
+                    "order"
+                ] is not None:
```

Comment:
just an else here (only way it gets is if `generator_name` not in the dictionary keys) and just `logging.warning`)

Reply:
The warning above is not needed anymore just because of the null columns now being accounted for.